src/App.vue

```diff
@@ -18,6 +17,79 @@ export default {
 <style>
 @import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@300;500;700&display=swap');
 
+:root {
```

Comment:
For this side, like you fit just add the "/* CSS HEX */" colors
The rest no really need, cause ebe the same thing expressed in different ways
And the "SCSS" ones no be valid CSS. Dem be part of a kind of superset of CSS called Sass.
Maybe dem fuck up the code for that side dah be why the variables no dey work.